shell/components/form/ArrayListGrouped.vue

```diff
@@ -106,10 +113,11 @@ export default {
       & > .remove {
         position: absolute;
 
-        padding: 0px;
+        padding: 0;
 
         top: 0;
         right: 0;
+        margin-right: -2.25em;
```

Comment:
I think since this component is used full-width elsewhere it might be better to do something like add more padding to `.info-box`